plaso/analysis/bloom.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+"""Analysis plugin to look up file hashes in bloom database
+
+"""
+try:
+  import flor
+except ImportError:
+  flor = None
+
+from plaso.analysis import hash_tagging
+from plaso.analysis import logger
+from plaso.analysis import manager
+
+
+class BloomAnalysisPlugin(hash_tagging.HashTaggingAnalysisPlugin):
+  """Analysis plugin for looking up hashes in bloom file."""
+
+  # hashlookup bloom files can handle a high load
+  # so look up all files.
+  DATA_TYPES = frozenset(['fs:stat', 'fs:stat:ntfs'])
+
+  NAME = 'bloom'
+
+  SUPPORTED_HASHES = frozenset(['sha1', 'md5', 'sha256'])
+
+  DEFAULT_LABEL = 'bloom_present'
+
+
+  def __init__(self):
+    """Initializes an hashlookup bloom  analysis plugin."""
+    super(BloomAnalysisPlugin, self).__init__()
+    self._label = self.DEFAULT_LABEL
+    self._bloom_database_path = None
+    self.bloom_filter_object = None
```

Comment:
@xmco why is bloom_filter_object a public class attribute?

Reply:
fixed in [296fa00](https://github.com/log2timeline/plaso/pull/4527/commits/296fa00ebe3109764f75b2c9adc2072c1d608fc7)